ruby/src/main/java/com/google/protobuf/jruby/RubyBuilder.java

```diff
@@ -0,0 +1,116 @@
+package com.google.protobuf.jruby;
```

Comment:
As a general comment -- can you put the copyright header at the top of each *.java file?